### ISSUE TITLE
CLI plugins requiring update output show as "FAILED"

### ISSUE THREAD

**Issue by coxd** [IBM-Cloud/ibm-cloud-developer-tools#120]
**Summary**

idt-install.sh does not deal with previously installed cli plugins that require updates - issues failed message.

**Steps to reproduce**
(include tooling version, platform, etc)
1. Have the ibmcloud CLI already installed
2. Have the container-service pulgin already installed
3. Run idt-install.sh

**Operating System**
Specify: MacOS

**Supporting details**

From the script output:

```[install_plugins] Installing plugin 'container-service'
Looking up 'container-service' from repository 'IBM Cloud'...
Plug-in 'container-service/kubernetes-service 0.3.8' found in repository 'IBM Cloud'
Plug-in 'container-service/kubernetes-service 0.2.102' was already installed. Do you want to update it with 'container-service/kubernetes-service 0.3.8' or not? [y/N]> 
FAILED
Could not read from input: EOF

```
-----------------------------------

**Development "done" checklist**

- [ ] Test case to verify
- [ ] Public Documentation updated
- [ ] Change added to "release notes" as appropropriate
- [ ] Notification to stakeholders (OM, other squads, etc)

**Comment by steveclay** [comment]
@coxd thanks for pointing this out. I believe we are going to modify the installer to just install all updates to the plugins going forward.

**Comment by steveclay** [comment]
Fixed by #122
